extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -552,6 +571,54 @@ export class ConsentUI {
     removeElement(dev().assertElement(this.ui_));
   }
 
+  /**
+   * If this is the first time viewing the iframe, create
+   * an 'invisible' alert dialog with a title and a button.
+   * Clicking on the button will transfer focus to the iframe.
+   */
+  maybeShowSrAlert_() {
+    if (this.restrictFullscreenOn_) {
+      // If this is not the first time seen, don't show it
+      if (this.srAlert_) {
+        return;
+      }
+
+      const alertDialog = this.document_.createElement('div');
+      const button = this.document_.createElement('button');
+      const titleDiv = this.document_.createElement('div');
+
+      alertDialog.setAttribute('role', 'alertdialog');
+
+      titleDiv.textContent = this.consentTitle_;
+      button.textContent = this.buttonTitle_;
+      button.onclick = () => {
+        this.ui_ = dev().assertElement(this.ui_);
+        tryFocus(this.ui_);
+      };
+
+      alertDialog.appendChild(titleDiv);
+      alertDialog.appendChild(button);
+
+      // Style to be visiblly hidden, but not hidden from the SR
+      const {classList} = alertDialog;
+      classList.add(consentUiClasses.screenReaderOnly);
+
+      this.baseInstance_.element.appendChild(alertDialog);
+      tryFocus(button);
+
+      this.srAlert_ = alertDialog;
+    }
+  }
+
+  /**
+   * Hide the SR alert if it exists.
+   */
+  maybeHideSrAlert_() {
+    if (this.srAlert_) {
```

Comment:
If I read it correctly, the `srAlert_` will only be shown once. If that's the case should we simply remove it from DOM and also unregister the event listener?

Reply:
Yes, good point. I will remove the element here. No listeners are attached.